conduit/pipeline/pipeline.go

```diff
@@ -57,6 +57,14 @@ type Config struct {
 	Processors []NameConfigPair `yaml:"processors"`
 	Exporter   NameConfigPair   `yaml:"exporter"`
 	Metrics    Metrics          `yaml:"metrics"`
+	// RetryCount is the number of retries to perform for an error in the pipeline
+	RetryCount uint64 `yaml:"retry-count"`
+	// RetryDelayString is the string that is converted into RetryDelay.  The format should follow the
+	// documentation laid out in the time.ParseDuration package
+	// As an example: retry-delay: 32ms would mean 32 milliseconds while 84h would mean 84 hours
+	RetryDelayString string `yaml:"retry-delay"`
```

Comment:
I believe yaml v3 supports time.Duration parsing, https://go.dev/play/p/-6y0zq96gVz

Reply:
Nice. I tried this with the json encoder a few days ago and it didn't: https://go.dev/play/p/BMc1fLrzAMe

Reply:
Done